scrapper.py

```diff
@@ -21,43 +40,160 @@ class IncorrectNumberOfArticlesError(Exception):
     """
 
 
+def _clean_text(text):
+    return re.sub(r"[\n\t ]+", " ", text).strip()
+
+
+def _get_page(link):
+    print("attempting to connect to %s", link)
```

Comment:
in the final version you should remove intermediate logging

Reply:
Removed